src/DynamoCoreWpf/Views/Core/AnnotationView.xaml

```diff
@@ -250,74 +250,188 @@
             <RowDefinition Height="*"></RowDefinition>
         </Grid.RowDefinitions>
         <Grid.ContextMenu>
-            <ContextMenu>
-                <MenuItem  Header="{x:Static p:Resources.GroupContextMenuDeleteGroup}"
-                           Name="DeleteAnnotation"
-                           Click="OnDeleteAnnotation" />
-                <MenuItem  Header="{x:Static p:Resources.GroupContextMenuUngroup}"
-                           Name="UngroupAnnotation"
-                           Click="OnUngroupAnnotation" />
-                <MenuItem Header="{x:Static p:Resources.ContextUnGroupFromSelection}"
-                          Name="unGroup_cm"
-                          Command="{Binding Path=RemoveGroupFromGroupCommand}" />
-                <MenuItem Header="{x:Static p:Resources.GroupContextMenuAddGroupToGroup}"
-                          Name="AddGroupToGroup"
-                          Command="{Binding AddGroupToGroupCommand}" />
-                <MenuItem  Header="{x:Static p:Resources.GroupContextMenuGraphLayout}"
-                           Name="GraphLayoutAnnotation"
-                           Click="OnGraphLayoutAnnotation" />
-                <MenuItem  Header="{x:Static p:Resources.GroupContextMenuFont}"
-                           Name="ChangeSize">
-                    <MenuItem Header="14"
-                              Name="FontSize0"
+            <ContextMenu Background="#666666" Style="{StaticResource ContextMenuStyle}">
+                <ContextMenu.Resources>
+                    <ResourceDictionary>
+                        <Style TargetType="{x:Type MenuItem}">
+                            <Setter Property="IsChecked" Value="{DynamicResource IsChecked}" />
+                            <!--<Setter Property="Height" Value="30" />-->
```

Comment:
Do we not need this?

Reply:
It was stopping the colours MenuIem from having a custom height

Reply:
So it will have auto height for that menu item?

Reply:
Yeah currently they're all Auto height. I could remove the commented code

Reply:
yeah, please remove